test/Spec.hs

```diff
@@ -76,6 +79,102 @@ allTests =
                           , ""
                           , "derive instance genericBar :: (Generic a, Generic b, Generic (m b)) => Generic (Bar a b m c)"
                           , ""
+                          , "--------------------------------------------------------------------------------"
+                          , "_Bar1 :: PrismP (Bar a b m c) (Maybe a)"
```

Comment:
Ah you're right - didn't notice as I don't have any Bar like types in my code (thankfully) :-) I'll push a fix.